src/libs/elektra/key.c

```diff
@@ -172,9 +187,7 @@ Key * keyNew (const char * name, ...)
 
 	if (!name)
 	{
-		k = (Key *) elektraMalloc (sizeof (Key));
-		if (!k) return 0;
-		keyInit (k);
+		k = elektraKeyMalloc ();
```

Comment:
Nice simplification!

Reply:
This is just rolling back my own changes :smile: